ui/src/app/views/resource-card/component.ts

```diff
@@ -190,9 +182,8 @@ export class ResourceCardComponent implements OnInit {
         break;
       }
       case ResourceType.NOTEBOOK: {
-        const nbUrl = '/workspaces/' + this.wsNamespace + '/' + this.wsId + '/notebooks/'
-          + encodeURIComponent(this.resourceCard.notebook.name);
-        window.open(nbUrl, '_blank');
+        this.route.navigate(['workspaces', this.wsNamespace, this.wsId, 'notebooks',
```

Comment:
I think this line can be reverted and included in your other PR (we still want to open a new tab for now)

Reply:
I swear I included this change in my other PR. I'll double check now and add it back in if it got left out somehow.